SYNTAX.md

```diff
@@ -203,6 +204,55 @@ postBuild {
 }
 ```
 
+### Triggers
+* *Description*:
+* *Required*: No
+* *Parameters*: None
+* *Takes a Closure*: Yes
+* *Closure Contents*: A sequence of one or more trigger configurations, using `@Symbol` names for constructors.
+    * Note that `[$class: 'Foo', arg1: 'something', ...]` syntax can not be used, only `cron('@daily')` and the like.
+    * Also note that the `SCMTrigger` won't work with the `scm` `@Symbol` - with Jenkins 2.22 or later, the `pollScm` symbol does work.
+* *Examples*:
+
+```
+triggers {
+    cron('@daily')
+}
+```
+
+### Build Parameters
+* *Description*:
+* *Required*: No
+* *Parameters*: None
+* *Takes a Closure*: Yes
+* *Closure Contents*: A sequence of one or more parameter definition configurations, using `@Symbol` names for constructors.
+    * Note that `[$class: 'Foo', arg1: 'something', ...]` syntax can not be used, only `booleanParam(...)` and the like.
+* *Examples*:
+
+```
+parameters {
+    booleanParam(defaultValue: true, description: '', name: 'flag')
+    string(defaultValue: '', description: '', name: 'SOME_STRING')
+}
+```
+
+### Job Properties
+* *Description*:
+* *Required*: No
+* *Parameters*: None
+* *Takes a Closure*: Yes
+* *Closure Contents*: A sequence of one or more job property configurations, using `@Symbol` names for constructors.
+    * Note that `[$class: 'Foo', arg1: 'something', ...]` syntax can not be used, only `booleanParam(...)` and the like.
+    * Note that the `parameters` and `pipelineTriggers` `@Symbol`s cannot be used here directly.
+* *Examples*:
+
+```
+jobProperties {
```

Comment:
this is what it is called elsewhere - it just looks funny here. If this is the accepted term, that is ok. Always confusing "properties" vs "parameters". Also we say "job" here but elsewhere say pipeline. Would "pipelineProperties" make more sense? settings?